### ISSUE TITLE
slot nums

### ISSUE THREAD

**Issue by wdyxwzyh** [dreasysnail/POINTER#13]
i didn't understand that for sequences {sos , x1, x2, ..., xt, eos},  shouldn't it be t+1 slots as described in INSERTION TRANSFORMER?  which means (sos ,x1) (x1,x2),...(xt, eos), but in your words, there is always t slots, which confused me a lot.

**Comment by wdyxwzyh** [comment]
{<s> , x1, x2, ..., xt, <e>}
 (<s> ,x1) (x1,x2),...(xt, <e>)

**Comment by dreasysnail** [comment]
Yes you are right. There should be t+1 slots. Could you point out where did you spot the error, so that we can update the paper/description? Thanks
